![BLAST sequence input window. Access BLAST from
<http://blast.ncbi.nlm.nih.gov/Blast.cgi>]( NucleotideBlast.png "fig:BLAST sequence input window. Access BLAST from http://blast.ncbi.nlm.nih.gov/Blast.cgi")
![Color-coded alignment
score]( BlastScores.png "fig:Color-coded alignment score") BLAST is the
abbreviation for Basic Local Alignment Search Tool. It can be accessed
via the NCBI (National Center for Biotechnological Information) website.
BLAST is one of the most widely used bioinformatics tools because it
helps researchers efficiently compare a query oligonucleotide or protein
sequence to large databases. Genes that share the highest homology with
the input sequence are identified, and thereby, the biological function
of the query gene is deduced. ![BLAST
result]( BLAST result.png "fig:BLAST result") BLAST uses statistics to
sort the sequence database and thus, identify the best matches. The user
receives the following output: Max score, Total Score, Query Coverage,
E-value, and Maximum Identity. The E-value or expected value represents
the number of hits you expect to find in a database of the same size.
Low E-values indicate a significant result. The Maximum Identity
represents the percentage of the shared identities between the query and
the hit in a nucleotide-to-nucleotide alignment. The alignment scores
are numerical values that describe the overall quality of an alignment.
The percentage of the query length that is included in the aligned
segments is called the query coverage.

[ ⇐ Next Generation Sequencing](NGS_Plant "wikilink")/ [ Phylogenetics
⇒](Phylogenetics "wikilink")\
[ ⇐ Theory Overview](PlantLab "wikilink")

