DNA isolation from cells can be performed using the phenol-chloroform
method. This method yields a genomic DNA that contains millions of
genes. In order to isolate a specific gene of interest from genomic DNA,
[PCR](PCR_MC "wikilink") using specific primers can be applied. If the
gene of interest has been provided in a vector, specific [restriction
enzyme](Restriction_enzyme "wikilink") can be used for the isolation.

Phenol-Chloroform Extraction
----------------------------

![ Phenol-chloroform extraction method is used for genomic DNA
isolation](phenol-chloroform.png "fig: Phenol-chloroform extraction method is used for genomic DNA isolation")
Nucleic acids from cells can be isolated using the phenol-chloroform
method. Phenol-chloroform is a mixture of buffer-saturated phenol and
chloroform in 1:1 ratio. The first step in the extraction is cell lysis
and homogenization in aqueous solution. Then the phenol-chloroform is
added to the mixture creating two phases that is further separated by
centrifugation. Purified phenol has a density of 1.07 g/cm^3^ while
chloroform has a higher density (1.47 g/cm^3^). Therefore,
centrifugation yields two phases: the lower organic phase and upper
aqueous phase.

Nucleic acids gain polarity from negatively charge phosphate backbone;
therefore, nucleic acids are soluble in upper aqueous phase. Proteins
contain hydrophobic regions that interact with phenol and cause the
proteins to precipitate at the interface between two phases (often as
white flocculation). Lipids also have a hydrophobic region and will
dissolve in the lower organic phase. The pH of the mixture determines
the separation of nucleic acids. In neutral pH, RNA and DNA are retained
in upper aqueous phase, but in acidic pH, DNA moves to the lower organic
phase and RNA remains in the upper aqueous phase. In the last step,
nucleic acids are recovered from the aqueous phase by precipitation
using isopropanol.

NanoDrop
--------

DNA yield (or DNA concentration) and purity can be determined by
measuring the absorbance using a spectrophotometer. NanoDrop is a
spectrophotometer that can measure absorbance from 0.5-2 μl by utilizing
sample retention system. Sample retention system enables the measurement
of samples with high concentration without dilution. Absorbance are
measured at 260nm (A260) where DNA absorbs light most strongly. The
amount of light absorbed is proportionate to the amount of DNA in the
sample.

The relationship is described by the Beer-Lambert Law: c = (A \* ε)/b\
c : the nucleic acid concentration in ng/μL\
A : the absorbance in AU\
ε : the wavelength-dependent extinction coefficient in ng-cm/μL\
b : the pathlength in cm\

-   Double-stranded DNA extinction coefficient is 50ng-cm/μL
-   In this case NanoDrop pathlength is 10 mm

DNA purity can be assessed by measuring it at λ = 280 nm, λ = 230 nm,
and λ = 320nm and calculating the ratio for 260/280nm , 260/230 nm, and
260/320 nm. Nucleic acids strongly absorb light at 260 nm while organic
contaminants such as phenol and Trizol absorb light at 230 nm and
proteins absorb light at 280 nm. Neither nucleic acids nor proteins
absorb light at 320 nm.

For 260/280, a ratio of approximately 1.8 is commonly accepted as pure
DNA while for pure RNA, the ratio is approximately 2. For 260/320, a
ratio 1.8-2.2 generally means that the nucleic acid sample is accepted
as pure. For 260/230, a ratio of \<1.8 represents a significant presence
of organic contaminants.

[Next (Polymerase Chain Reaction) ⇒](PCR_MC "wikilink")\
[⇐ Theory overview](Molecular Cloning "wikilink")

