<html><head></head><body><div lang="en" dir="ltr" class="mw-content-ltr"><p>DNA isolation from cells can be performed using the phenol-chloroform method. This method yields a genomic DNA that contains millions of genes. In order to isolate a specific gene of interest from genomic DNA, <a href="/index.php/PCR_MC" title="PCR MC">PCR</a> using specific primers can be applied. If the gene of interest has been provided in a vector, specific <a href="/index.php/Restriction_enzyme" title="Restriction enzyme">restriction enzyme</a> can be used for the isolation.
</p><p><br/>
</p>
<h2> <span class="mw-headline" id="Phenol-Chloroform_Extraction"> Phenol-Chloroform Extraction</span></h2>
<div class="thumb tright"><div class="thumbinner" style="width:402px;"><a href="/index.php/File:Phenol-chloroform.png" class="image"><img alt="" src="/images/thumb/Phenol-chloroform.png/400px-Phenol-chloroform.png" width="400" height="258" class="thumbimage"/></a>  <div class="thumbcaption"><div class="magnify"><a href="/index.php/File:Phenol-chloroform.png" class="internal" title="Enlarge"><img src="/skins/common/images/magnify-clip.png" width="15" height="11" alt=""/></a></div>Phenol-chloroform extraction method is used for genomic DNA isolation</div></div></div>
<p>Nucleic acids from cells can be isolated using the phenol-chloroform method. Phenol-chloroform is a mixture of buffer-saturated phenol and chloroform in 1:1 ratio. The first step in the extraction is cell lysis and homogenization in aqueous solution. Then the phenol-chloroform is added to the mixture creating two phases that is further separated by centrifugation. Purified phenol has a density of 1.07 g/cm<sup>3</sup>  while chloroform has a higher density (1.47 g/cm<sup>3</sup>). Therefore, centrifugation yields two phases: the lower organic phase and upper aqueous phase.
</p><p><br/>
Nucleic acids gain polarity from negatively charge phosphate backbone; therefore, nucleic acids are soluble in upper aqueous phase. Proteins contain hydrophobic regions that interact with phenol and cause the proteins to precipitate at the interface between two phases (often as white flocculation). Lipids also have a hydrophobic region and will dissolve in the lower organic phase. The pH of the mixture determines the separation of nucleic acids. In neutral pH, RNA and DNA are retained in upper aqueous phase, but in acidic pH, DNA moves to the lower organic phase and RNA remains in the upper aqueous phase. In the last step, nucleic acids are recovered from the aqueous phase by precipitation using isopropanol.
</p><p><br/>
</p>
<h2> <span class="mw-headline" id="NanoDrop">NanoDrop</span></h2>
<p>DNA yield (or DNA concentration) and purity can be determined by measuring the absorbance using a spectrophotometer. NanoDrop is a spectrophotometer that can measure absorbance from 0.5-2 &#956;l by utilizing sample retention system. Sample retention system enables the measurement of samples with high concentration without dilution.  Absorbance are measured at 260nm (A260) where DNA absorbs light most strongly. The amount of light absorbed is proportionate to the amount of DNA in the sample. 
</p><p><br/>
The relationship is described by the Beer-Lambert Law: 
c = (A * &#949;)/b <br/>
c&#160;: the nucleic acid concentration in ng/&#956;L <br/>
A&#160;: the absorbance in AU <br/>
&#949;&#160;: the wavelength-dependent extinction coefficient in ng-cm/&#956;L <br/>
b&#160;: the pathlength in cm <br/>
</p>
<ul><li>Double-stranded DNA extinction coefficient is 50ng-cm/&#956;L
</li><li>In this case NanoDrop pathlength is 10 mm
</li></ul>
<p><br/>
DNA purity can be assessed by measuring it at &#955; = 280 nm, &#955; = 230 nm, and &#955; = 320nm and calculating the ratio for 260/280nm , 260/230 nm, and 260/320 nm.
Nucleic acids strongly absorb light at 260 nm while organic contaminants such as phenol and Trizol absorb light at 230 nm and proteins absorb light at 280 nm. Neither nucleic acids nor proteins absorb light at 320 nm.
</p><p><br/>
For 260/280, a ratio of approximately 1.8 is commonly accepted as pure DNA while for pure RNA, the ratio is approximately 2. For 260/320, a ratio 1.8-2.2 generally means that the nucleic acid sample is accepted as pure. For 260/230, a ratio of &lt;1.8 represents a significant presence of organic contaminants.
</p><p><br/>
<a href="/index.php/PCR_MC" title="PCR MC">Next (Polymerase Chain Reaction) &#8658;</a> <br/>
<a href="/index.php/Molecular_Cloning" title="Molecular Cloning">&#8656; Theory overview</a>
</p>
<!-- 
NewPP limit report
Preprocessor node count: 7/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->

<!-- Saved in parser cache with key labster_com_db-Learn_:pcache:idhash:3981-0!*!0!!*!2!* and timestamp 20140813023122 -->
</div>				</body></html>