<html><head></head><body><div lang="en" dir="ltr" class="mw-content-ltr"><div class="thumb tright"><div class="thumbinner" style="width:402px;"><a href="/index.php/File:Gene_structure.png" class="image"><img alt="" src="/images/thumb/Gene_structure.png/400px-Gene_structure.png" width="400" height="63" class="thumbimage"/></a>  <div class="thumbcaption"><div class="magnify"><a href="/index.php/File:Gene_structure.png" class="internal" title="Enlarge"><img src="/skins/common/images/magnify-clip.png" width="15" height="11" alt=""/></a></div>Gene structure</div></div></div> Plants produce numerous secondary metabolites that hold tremendous potential for various applications such as pharmaceuticals, flavors, dyes, fragrances and insecticides. However, the plant&#8217;s secondary metabolism remains poorly characterized. In order to take advantage of nature's biosynthetic machinery, knowledge of the enzymatic pathways and the underlying genetic maps are essential. The emergence of Next Generation Sequencing (NGS) made it possible to affordably sequence millions of DNA sequences. Therefore, the amount of bioinformatics data is growing exponentially. Functional genomics uses this vast amount of data to investigate the function of genes at the genomic, transcriptomic and proteomic level. In this lab, we will use functional genomics techniques to characterize the enzymatic pathway of a terpenoid, which has the potential to be used as an antimalarial drug.
<br/><div class="thumb tleft"><div class="thumbinner" style="width:402px;"><a href="/index.php/File:Genetic_code.png" class="image"><img alt="" src="/images/thumb/Genetic_code.png/400px-Genetic_code.png" width="400" height="517" class="thumbimage"/></a>  <div class="thumbcaption"><div class="magnify"><a href="/index.php/File:Genetic_code.png" class="internal" title="Enlarge"><img src="/skins/common/images/magnify-clip.png" width="15" height="11" alt=""/></a></div>Genetic code <br/>Each codon (nucleotide triplet) encodes for an amino acid.</div></div></div>
<p><b>Transcription</b><br/>
Each enzyme consists of one or several amino acid chains that fold into the three dimensional structure of a protein. Each amino acid is encoded by three nucleotides of the corresponding gene sequence. The genes are stretches of DNA molecules contained within the cell nucleus. A complex network of signaling molecules and transcription factors regulates gene transcription in each cell. When protein complexes bind to the promoter sequence it triggers the transcription of the DNA code into a complementary RNA sequence.<br/>
Watch this animation for a molecular representation of the transcription process: <a rel="nofollow" target="_blank" class="external free" href="http://www.youtube.com/watch?v=5MfSYnItYvg">http://www.youtube.com/watch?v=5MfSYnItYvg</a>
</p><p><br/>
<b>RNA Processing</b><br/>
Once transcribed, the single-stranded RNA molecule folds back on itself and is further processed by enzyme complexes. RNA molecules perform multiple vital roles. Only about 3% of total plant RNAs encode for proteins. These so called messenger RNAs are processed by the spliceosome, which splices out certain portions of the RNA. These spliced fragments are called introns whereas the remaining protein coding sequence consists of the exons. Introns could be considered unimportant, junk sequences; however, they are also responsible for alternative splicing patterns, which leads to different proteins being derived from a single gene. Some introns are also processed to become functional, non-coding RNAs. <br/>
Multiple adenine residues are added to the 3' end and methylated guanine are added to the 5' end. The remaining stretch of exons with the flanking un-translated regions (UTR), the mature mRNA, is exported out of the nucleus. <br/>
Watch this animation for a molecular representation of the splicing machinery: <a rel="nofollow" target="_blank" class="external free" href="http://www.youtube.com/watch?v=aVgwr0QpYNE">http://www.youtube.com/watch?v=aVgwr0QpYNE</a>
</p><p><br/>
<b>Translation</b><br/>
Upon the recruitment to a ribosome, the mRNA is translated into the amino acid sequence. Translation is initiated at the start codon, which marks the beginning of the exon sequence and determines the reading frame for the translation. Following the start codon, Adenosine-Uracil-Guanine (AUG), each nucleotide triplet is translated into an amino acid until a stop codon (UAA, UAG or UGA) terminates the translation. The newly synthesized peptide chain folds back on itself and forms the three dimensional structure of the enzyme.<br/>
Watch this animation for a molecular representation of the translation machinery: <a rel="nofollow" target="_blank" class="external free" href="http://www.youtube.com/watch?v=8dsTvBaUMvw">http://www.youtube.com/watch?v=8dsTvBaUMvw</a> 
<br/><br/>
</p>
<h3> <span class="mw-headline" id="Transcriptomics"> Transcriptomics </span></h3>
<p>Protein sequencing is much more difficult than DNA sequencing. Therefore, the protein sequences are usually obtained from the oligonucleotide sequence that encodes the proteins. To study the enzymes in a certain cell population it is not necessary to sequence the whole genome of the organism. The transcriptome is the set of all transcribed RNA molecules in one, or a population of, cells. Because the transcriptome only comprises a tiny fraction of the genome, transcriptome sequencing is much faster and only intrinsically sequences the transcribed genes. In addition, the transcriptomic data provides information about the quantity of gene expression, which can be used to analyze differential gene expression. In a living cell there is a constant turnover of RNA and gene expression is continuously adjusted. The instability of RNA molecules makes it necessary to reverse transcribe the RNA into DNA for sequencing.
</p><p><br/>
</p><p><a href="/index.php/Killer_Assay" title="Killer Assay"> &#8656; Killer Assay </a>/ <a href="/index.php/CDNA" title="CDNA"> Next cDNA &#8658;</a> <br/>
<a href="/index.php/PlantLab" title="PlantLab"> &#8656; Theory Overview </a>
</p>
<!-- 
NewPP limit report
Preprocessor node count: 3/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->

<!-- Saved in parser cache with key labster_com_db-Learn_:pcache:idhash:4109-0!*!0!*!*!2!* and timestamp 20140813023246 -->
</div>				</body></html>