<html><head></head><body><div lang="en" dir="ltr" class="mw-content-ltr"><p>We can perform genetic testing to determine whether the trait is inherited in the family. In 1990, Mary-Claire King from UC Berkeley performed linkage analysis in families that have breast cancer and narrowed down the location of the gene to chromosome 17q21  <a rel="nofollow" target="_blank" class="external autonumber" href="http://www.ncbi.nlm.nih.gov/pubmed/2270482%7CPubMed">[1]</a>. This gene was later named as <i>BRCA1</i>. 
</p><p><br/>
</p>
<h2> <span class="mw-headline" id="Linkage_analysis">Linkage analysis</span></h2>
<div class="thumb tleft"><div class="thumbinner" style="width:202px;"><a href="/index.php/File:HR.png" class="image"><img alt="" src="/images/thumb/HR.png/200px-HR.png" width="200" height="300" class="thumbimage"/></a>  <div class="thumbcaption">Figure 1. Homologous recombination during meiosis giving rise to new genetic variations.</div></div></div>
<p>Genes that are located in proximity with each other tend to be inherited together during meiosis. During meiosis, the two sister chromatids are separated and crossing over can occur, resulting in homologous recombination, where material from one sister chromatid is transferred to the other chromatid. This event gives rise to genetic variations that separate one individual from the others. As shown in Figure 1, when two genes are located close to each other, they have a high chance of being inherited together because they are both exchanged during homologous recombination. On the other hand, if two genes are located at opposite ends of the chromosome, they will not be inherited together. One gene will be included in the homologous recombination and transferred into one <span style="color:#FF4500">sister chromatids</span> while the other gene will stay in the <span style="color:#0000FF">original chromatids</span>. Because only one pair of chromatids (one chromosome) is passed to the offspring and the two genes are now located in two different chromosomes, the offspring will only inherit one of the two genes. 
</p><p><br/>
Linkage analysis is very useful to determine the approximate location of an interesting gene, for example, one that is suspected to cause a disease or condition. To identify this gene, we can use markers or specific DNA sequences with known exact locations. For example, we can use microsatellites or SNP (single nucleotide polymorphism) as markers. By observing which allele variation is always present in the affected individual, we can then narrow down the approximate location of the gene. 
</p><p><br/>
We need to analyze many individuals in the family over several generations to identify the gene location because we need to find a specific allele that is always present in the affected individual and never present in the healthy individual. There are less genetic variations within a family because they are all genetically related. This is why it is best to compare individuals within a close family rather than finding a healthy individual from an unrelated family. A healthy individual may have the same allele. However, because an individual comes from an unrelated family, that individual does not possess the disease gene. Therefore, we always need to compare within a close family. 
</p><p><br/>
</p>
<h2> <span class="mw-headline" id="Microsatellite">Microsatellite</span></h2>
<p>Microsatellites are short repeating DNA sequences of two to six base pairs. The numbers of repeats can vary among individuals; they represent the different alleles. For example, allele 1 has four repeats, allele 2 has five repeats, allele 3 has six repeats, and so forth. SNPs can also be used as markers in linkage analysis; however, there are only two alleles for SNPs. For example, C/T, which means that allele 1 has C, while allele 2 has T. Due to the higher number of variability in microsatellites than in SNPs, it is easier to perform linkage analysis using microsatellites.  
</p><p><br/>
One of the most common examples of microsatellite is (CA)<sub>n</sub>, where <sub>n</sub> represents the number of CA repeats. The number of repeats can vary between three to 100 times, creating a high number of alleles that are present in the population. Remember that this high number of alleles is present in the population and not in the individual! We only have two copies of chromosomes meaning that we only have maximum two different alleles present in our cells. When an individual has two different alleles, for example (CA)<sub>10</sub> and (CA)<sub>15</sub> he/she is said to be heterozygous for the microsatellite. On the other hand, when an individual has only one allele in both the chromosomes, he/she is said to be homozygous for the microsatellite. We can analyze the genotype of the microsatellite by amplifying it using <a href="/index.php/PCR_MG" title="PCR MG">PCR</a> and analyzing the length (the number of repeats) using <a href="/index.php/Gel_electrophoresis_MG" title="Gel electrophoresis MG">gel electrophoresis</a>. 
</p><p><br/>
</p>
<h2> <span class="mw-headline" id="Reference">Reference</span></h2>
<p>1. Hall JM, Lee MK, Newman B, Morrow JE, Anderson LA, Huey B, King MC., 1990, <i>Linkage of early-onset familial breast cancer to chromosome 17q21.</i> Science Dec 21;250(4988):1684-9.
</p><p><br/>
<a href="/index.php/PCR_MG" title="PCR MG">Next (PCR) &#8658;</a>
</p><p><a href="/index.php/Medical_Genetics_case" title="Medical Genetics case">&#8656; Theory overview</a>
</p>
<!-- 
NewPP limit report
Preprocessor node count: 13/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->

<!-- Saved in parser cache with key labster_com_db-Learn_:pcache:idhash:3958-0!*!0!!*!2!* and timestamp 20140812082642 -->
</div>				</body></html>