<html><head></head><body><div lang="en" dir="ltr" class="mw-content-ltr"><p>DNA sequencing is a method to identify each base that makes up the DNA strand. 
</p><p><br/>
</p>
<h2> <span class="mw-headline" id="First_Generation_Sequencing">First Generation Sequencing</span></h2>
<div class="thumb tright"><div class="thumbinner" style="width:283px;"><a href="/index.php/File:Sangersequencing.png" class="image"><img alt="" src="/images/thumb/Sangersequencing.png/281px-Sangersequencing.png" width="281" height="400" class="thumbimage"/></a>  <div class="thumbcaption"><b>Figure 1.</b> Chain-termination method. The labelled DNA molecules are separated according their size. Each molecule has a modified nucleotide (ddNTP) that is labeled with a fluorescence dye. The different fluorescence color indicates a specific base:  <span style="color:#00ff00">A</span>, <span style="color:#ffd700">G</span>, <span style="color:#ff0000">T</span> or <span style="color:#0000ff">C</span>.</div></div></div>
<p>First-generation sequencing refers to the chain-termination method that was developed by Fredrick Sanger and co-workers in 1977 (Figure 1). The DNA molecule is amplified with a modified nucleotide, allowing only one addition of base per cycle. DNA is then amplified with varying length: one strand having one base pair longer than the previous molecule. These molecules are then separated in a capillary. Each base is labeled with a fluorescence dye. By reading the different color signal, we will be able to identify the base (A, G, T, or C). 
</p>
<h2> <span class="mw-headline" id="Next_Generation_Sequencing">Next Generation Sequencing</span></h2>
<p>Next-generation sequencing is an advanced sequencing technology where many short DNA molecules are sequenced at the same time. The technology is also called massively parallel sequencing. These short DNA molecules are then assembled by comparing their sequence to a reference sequence, thereby revealing the complete DNA sequence that can be very long (as long as our genome!). 
</p><p><br/>
There are many different platforms of next-generation sequencing, and each of them utilize a different technique in achieving DNA sequencing. In this case, we will focus on the <i>reversible dye termination technology</i> employed by Illumina. In this method, DNA is first fragmented into a smaller size, and two short DNA molecules called "Adapters" are ligated to each end of the sample (Figure 2). These adapters will function as a primer-docking site to amplify DNA during PCR and to bind to the flow cell. Before analyzing the data, we remove the adapters because they are not a biological sequence. 
</p><p><br/>
DNA molecules capped with adapters and primers are first attached on a slide (called the flow cell) and amplified with polymerase enzyme creating local clonal DNA colonies. These DNA colonies are also referred to as <i>DNA clusters</i>. Each cluster contains exactly the same DNA sequence; therefore the term clonal DNA colonies. Similar to the first-generation Sequencing technique, the nucleotides are individually labeled with a fluorescence dye. After the addition of one nucleotide, the elongation stops and a picture is taken. Following this, the blocker sitting on the 3' terminal is then chemically removed from DNA, allowing the next cycle of nucleotide addition to proceed.
</p>
<div class="thumb tleft"><div class="thumbinner" style="width:760px;"><a href="/index.php/File:Illumina.jpg" class="image"><img alt="" src="/images/Illumina.jpg" width="758" height="640" class="thumbimage"/></a>  <div class="thumbcaption"><b>Figure 2.</b> Workflow of next-generation sequencing using Illumina platform.</div></div></div>
<p><br/><br/><br/><br/><br/><br/><br/>
The parallel sequencing produces millions and billions of reads per run. This is exponentially larger than the reads produced by first-generation sequencing. Next-generation sequencing is a very powerful technique that can be used for many different applications, such as SNP profiling, gene expression analysis, and detecting genetic aberrations such as mutation or chromosomal re-arrangements. Once DNA is extracted, it needs to be processed in preparation for sequencing. 
</p><p><br/>
The different sample preparation steps are outlined below:
</p>
<ul><li> Fragmentation
</li><li> End-repair
</li><li> A-tailing
</li><li> Adapter ligation
</li><li> PCR amplification
</li></ul>
<p>This is followed by cluster generation and the actual sequencing process.  
</p><p><br/>
</p>
<h2> <span class="mw-headline" id="DNA_mutation">DNA mutation</span></h2>
<p>There are several classifications for DNA mutations; you can find out more about the mutation information from the way it is written. For example: 
</p>
<ul><li>345G&gt;T means that there is one nucleotide substitution from guanine to thymine on the position 345. 
</li><li>345delGT means that there is deletion of two nucleotides (guanine and thymine) starting from position 345. 
</li><li>345dupA means that there is a duplication in position 345, resulting in two adenine residues.
</li><li>345insTA means that there are two nucleotides inserted (thymine and adenine) starting from position 345. 
</li></ul>
<p><br/>
</p><p><br/>
<a href="/index.php/Protein_Truncation_Test" title="Protein Truncation Test">&#8656; Previous (Protein Truncation Test)</a>
</p><p><a href="/index.php/Medical_Genetics_case" title="Medical Genetics case">&#8656; Theory overview</a>
</p>
<!-- 
NewPP limit report
Preprocessor node count: 15/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->

<!-- Saved in parser cache with key labster_com_db-Learn_:pcache:idhash:3963-0!*!0!!*!2!* and timestamp 20140812082850 -->
</div>				</body></html>