<html><head></head><body><div lang="en" dir="ltr" class="mw-content-ltr"><div class="thumb tleft"><div class="thumbinner" style="width:302px;"><a href="/index.php/File:IlluminaNGSflow.png" class="image"><img alt="" src="/images/thumb/IlluminaNGSflow.png/300px-IlluminaNGSflow.png" width="300" height="364" class="thumbimage"/></a>  <div class="thumbcaption"><div class="magnify"><a href="/index.php/File:IlluminaNGSflow.png" class="internal" title="Enlarge"><img src="/skins/common/images/magnify-clip.png" width="15" height="11" alt=""/></a></div>Workflow of Next Generation Sequencing using Illumina platform.</div></div></div>
<h3> <span class="mw-headline" id="Next_Generation_Sequencing">Next Generation Sequencing</span></h3>
<p>Next Generation Sequencing is an advanced sequencing technology that sequences millions of short DNA molecules at the same time. The technology is also called massive parallel sequencing. 
</p><p>There are many different Next Generation Sequencing platforms: each of them utilizes a different technique. In this case, we will focus on the reversible dye termination technology employed by Illumina. First, the DNA is fragmented and the two short DNA molecules, called adapters, are ligated to each end of each sequence (see Figure 2). These adapters function as primer-docking sites for the PCR and attach to flow cells. 
</p><p>DNA molecules capped with adapters are first attached to a slide (called the flow cell) and are then amplified by the polymerase enzyme creating local clonal DNA colonies. These DNA colonies are also referred to as DNA clusters. Each of the clusters contains exactly the same DNA sequence, hence the term clonal DNA colonies. Similar to the First Generation Sequencing technique, the nucleotides are individually labeled with a fluorescence dye. After the addition of one nucleotide, the elongation stops and a picture is taken. Thereafter, the blocker, sitting at the 3' terminus, is chemically removed from the DNA allowing the next cycle of nucleotide addition to occur.
</p><p><br/>
Once the DNA is extracted, it needs to be processed in preparation for sequencing. The different sample preparation steps are outlined below:
</p>
<ul><li> Fragmentation
</li><li> End-repair
</li><li> A-tailing
</li><li> Adapter ligation
</li><li> PCR amplification
</li><li> Followed by Cluster generation and the actual Sequencing process. 
</li></ul>
<p><br/>
The reads are assembled by comparing their sequence to a reference sequence. Overlaps of reads allow for the reconstruction of the original DNA fragments.
</p><p>Parallel sequencing produces millions of reads per run. This is exponentially larger than the reads that are produced by First Generation Sequencing. Next Generation Sequencing is a very powerful technique that can be used for many different applications, such as SNP profiling, gene expression analysis and the detection of genetic aberrations (e.g. mutations or chromosomal rearrangements).<br/>
For this case we want to identify protein-coding sequences; therefore we can exclude all of the RNA species other than mRNAs. Relevant sequences contain a start codon, followed by several hundred codons, and then a stop codon.
</p><p><br/>
<a href="/index.php/CDNA" title="CDNA"> &#8656; cDNA</a>/ <a href="/index.php/BLAST" title="BLAST"> BLAST &#8658;</a><br/>
<a href="/index.php/PlantLab" title="PlantLab">&#8656; Theory overview</a>
</p>
<!-- 
NewPP limit report
Preprocessor node count: 3/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->

<!-- Saved in parser cache with key labster_com_db-Learn_:pcache:idhash:4100-0!*!0!*!*!2!* and timestamp 20140812082852 -->
</div>				</body></html>