<html><head></head><body><div lang="en" dir="ltr" class="mw-content-ltr"><p>DNA sequencing is a method to identify each base that makes up the DNA strand. 
</p><p><br/>
</p>
<table id="toc" class="toc"><tr><td><div id="toctitle"><h2>Contents</h2></div>
<ul>
<li class="toclevel-1 tocsection-1"><a href="#First_Generation_Sequencing"><span class="tocnumber">1</span> <span class="toctext">First Generation Sequencing</span></a></li>
<li class="toclevel-1 tocsection-2"><a href="#Next_Generation_Sequencing"><span class="tocnumber">2</span> <span class="toctext">Next Generation Sequencing</span></a></li>
<li class="toclevel-1 tocsection-3"><a href="#How_to_extract_the_DNA.3F"><span class="tocnumber">3</span> <span class="toctext">How to extract the DNA?</span></a></li>
<li class="toclevel-1 tocsection-4"><a href="#Magnetic_beads"><span class="tocnumber">4</span> <span class="toctext">Magnetic beads</span></a></li>
</ul>
</td></tr></table>
<h3> <span class="mw-headline" id="First_Generation_Sequencing">First Generation Sequencing</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:283px;"><a href="/index.php/File:Sangersequencing.png" class="image"><img alt="" src="/images/thumb/Sangersequencing.png/281px-Sangersequencing.png" width="281" height="400" class="thumbimage"/></a>  <div class="thumbcaption"><b>Figure 1.</b> Chain-termination method. The labeled DNA molecules are separated according to their size. Each of the molecule has a modified nucleotide (ddNTP) that is labeled with a fluorescence dye. The different fluorescence color indicates a specific base,i.e., <span style="color:#00ff00">A</span>, <span style="color:#ffd700">G</span>, <span style="color:#ff0000">T</span> or <span style="color:#0000ff">C</span>.</div></div></div>
<p>The first generation sequencing refers to the <i>chain-termination method</i> that was developed by Fredrick Sanger and co-workers in 1977 (see Figure 1). The DNA molecule is amplified with a modified nucleotide allowing only 1 addition of base per cycle. The DNA is then amplified with varying length, 1 strand having 1 base pair longer than the previous molecule. These molecules are then separated in a capillary electrophoresis. Each of the bases is labeled with fluorescence dye. By reading the different color signal, we will be able to identify the bases, i.e., A, G, T or C. 
</p><p><br/>
</p>
<h3> <span class="mw-headline" id="Next_Generation_Sequencing">Next Generation Sequencing</span></h3>
<p>Next Generation Sequencing is an advance sequencing technology where many short DNA molecules are sequenced at the same time. The technology is also called massively parallel sequencing. These short DNA molecules are then assembled by comparing their sequence to a reference sequence, thereby revealing the complete DNA sequence that can be very long (as long as our genome!).
</p><p><br/>
There are many different platforms of Next Generation Sequencing, each of them utilize a different technique in achieving DNA sequencing. In this case, we will focus on the <i>reversible dye termination technology</i> employed by Illumina. In this method the DNA is first fragmented into smaller sizes and 2 short DNA molecules called "Adapters" are ligated to each end of the sample (see Figure 2). These adapters will function as primer-docking site to amplify the DNA during PCR and to bind to the flow cell. Before analyzing the data, we remove the adapters because they are not a biological sequence. 
</p><p><br/>
DNA molecules capped with adapters and primers are first attached on a slide (called the flow cell) and amplified with the polymerase enzyme creating local clonal DNA colonies. These DNA colonies are also referred to as <i>DNA clusters</i>. Each of the cluster contains exactly the same DNA sequence, hence the term clonal DNA colonies. Similar to the First Generation Sequencing technique, the nucleotides are individually labeled with a fluorescence dye. After the addition of 1 nucleotide, the elongation stops and a picture is taken. Following which, the blocker sitting at the 3' terminal is then chemically removed from the DNA allowing the next cycle of nucleotide addition to proceed.
</p><p><br/>
Once the DNA is extracted, it needs to be processed in preparation for sequencing. The different sample preparation steps are outlined below:
</p>
<ul><li> <a href="/index.php/Fragmentation" title="Fragmentation">Fragmentation</a>
</li><li> <a href="/index.php/End-repair" title="End-repair">End-repair</a>
</li><li> <a href="/index.php/A-tailing" title="A-tailing">A-tailing</a>
</li><li> <a href="/index.php/Adapter_ligation" title="Adapter ligation">Adapter ligation</a>
</li><li> <a href="/index.php/PCR_amplification" title="PCR amplification">PCR amplification</a>
</li></ul>
<p>Followed by <a href="/index.php/Cluster_generation" title="Cluster generation">Cluster generation</a> and the actual <a href="/index.php/Sequencing_process" title="Sequencing process">Sequencing process</a>. 
</p>
<div class="thumb tleft"><div class="thumbinner" style="width:414px;"><a href="/index.php/File:IlluminaNGSflow.png" class="image"><img alt="" src="/images/thumb/IlluminaNGSflow.png/412px-IlluminaNGSflow.png" width="412" height="500" class="thumbimage"/></a>  <div class="thumbcaption"><b>Figure 2.</b> Workflow of Next Generation Sequencing using Illumina platform.</div></div></div>
<p>The parallel sequencing produces millions and billions of reads per run. This is exponentially larger than the reads that are produced by First Generation Sequencing. Next Generation Sequencing is a very powerful technique that can be used for many different applications, such as SNP profiling, gene expression analysis, detecting genetic aberrations such as mutation or chromosomal rearrangements. 
</p><p><br/>
</p>
<h3> <span class="mw-headline" id="How_to_extract_the_DNA.3F">How to extract the DNA?</span></h3>
<p>DNA can be extracted from any cell. First, we need to disrupt the cell (also known as cell lysis). This can be achieved by using chemical and physical methods, grinding or sonicating the cell. Once the cell is disrupted, we need to remove the membrane lipids by adding surfactants or detergents, digest the protein using protease and remove the RNA by using RNase. Following which, we can precipitate the DNA (for example, using ethanol) and isolate the DNA from all other materials present in a cell. There are several methods that can be chosen to extract the DNA, the most popular method is by using phenol and chloroform. However, new methods that are easier and safer have been developed, for example, using column separation. 
</p><p>Please note that extracting DNA from a bone sample can be quite tricky as compared to blood samples or tissue samples. There are many commercial kits available for extracting specific sample including for bone samples.  
</p><p><br/>
</p>
<h3> <span class="mw-headline" id="Magnetic_beads">Magnetic beads</span></h3>
<p>During the NGS sample preparation, the DNA needs to be purified from other materials several times. There are various methods that can be used to achieve this. For example, using column separation (similar to the DNA extraction method) or using magnetic beads, such as in this case. The magnetic beads will bind selectively to the DNA molecules when placed under the magnetic rack, and all the beads will be separated from the solution. We can then pipette the solution, containing all the unwanted materials (such as the end-repair mix, A-tailing mix, and other) and wash the beads using 80% ethanol. After several rounds of washing, we can elute the DNA molecules from the magnetic beads and transfer them to a new tube.
</p><p><br/>
</p><p><a href="/index.php/Fragmentation" title="Fragmentation">Next (Fragmentation) &#8658;</a>
</p><p><a href="/index.php/NGS_Case" title="NGS Case">&#8656; Theory overview</a>
</p>
<!-- 
NewPP limit report
Preprocessor node count: 19/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->

<!-- Saved in parser cache with key labster_com_db-Learn_:pcache:idhash:3836-0!*!0!!en!2!* and timestamp 20140814080717 -->
</div>				</body></html>