<html><head></head><body><div lang="en" dir="ltr" class="mw-content-ltr"><div class="thumb tright"><div class="thumbinner" style="width:442px;"><a href="/index.php/File:Phylogenetic_tree.gif" class="image"><img alt="" src="/images/Phylogenetic_tree.gif" width="440" height="243" class="thumbimage"/></a>  <div class="thumbcaption">Phylogenetic tree of terpenoid synthase (Bohlmann, 1998. doi:10.1073/pnas.95.8.4126)</div></div></div>
<p>Phylogenetics is the study of evolutionary relationship between biological entities. It can be used to determine the eevolutionary distance of two DNA sequences as well as for whole organisms. Phylogenetic analysis is based on the comparison of conserved hereditary sequences. In our case we want to study the phylogenetic relationship of different enzymes. In the case of proteins one could align the amino acid sequence as well as the underlying nucleotide coding sequence. However the functionality of a protein is better reflected in the amino acid sequence. A point mutation in the DNA sequence does not necessarily lead to a change in the peptide chain because several different codons are translated into the same amino acid. Conversely a one nucleotide insertion might look like a minor change on the DNA level, on the protein level however it shifts the whole codon pattern and therefore the sequence is translated into a completely different amino acid sequence. Therefore the amino acid sequences of the different proteins are compared with each other.
The seqences are aligned such that they reach the maximum similarity. The amino acid replacements are then rated according to their likelihood. For example a Gly-Ala change is more likely than a Gly-Pro change which would alter the 3D structure of the protein. Thereby we can derive the degree of homology. These evolutionary distances are usually depicted as a phylogenetic tree. To validate the result, the analysis is repeated several times and the resulting trees are superimposed on each other (bootstrapping), showing only the one which is most likely. Each node gets a value assigned based on the proportion of bootstrap trees showing that same clade. Nodes with a value &gt;70% are considered as consistent. <br/>
The resulting phylogenetic gives insights on the taxonomy of the different species. In the case of proteins the close homologs are usually responsible for similar functions. By including proteins with known functions into the phylogenetic analysis one can derive the role of other enzymes with a high degree of homology.
</p><p><br/>
</p><p><a href="/index.php/BLAST" title="BLAST">&#8656; BLAST</a>/ <a href="/index.php/Moss_Transformation" title="Moss Transformation"> Moss Transformation &#8658;</a><br/>
<a href="/index.php/PlantLab" title="PlantLab"> &#8656; Theory Overview</a>
</p>
<!-- 
NewPP limit report
Preprocessor node count: 1/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->

<!-- Saved in parser cache with key labster_com_db-Learn_:pcache:idhash:4106-0!*!0!*!*!2!* and timestamp 20140813094950 -->
</div>				</body></html>