<html><head></head><body><div lang="en" dir="ltr" class="mw-content-ltr"><div class="thumb tright"><div class="thumbinner" style="width:442px;"><a href="/index.php/File:Phylogenetic_tree.gif" class="image"><img alt="" src="/images/Phylogenetic_tree.gif" width="440" height="243" class="thumbimage"/></a>  <div class="thumbcaption">Phylogenetic tree of terpenoid synthase (Bohlmann, 1998. doi:10.1073/pnas.95.8.4126)</div></div></div> 
<h3> <span class="mw-headline" id="Basic_principle"> Basic principle </span></h3>
<p>Phylogenetics is the study of the evolutionary relationship between biological entities. It can be used to determine the evolutionary distance of DNA sequences or entire organisms. Phylogenetic analysis is based on the comparison of conserved hereditary sequences.  <br/>
</p><p><br/>
</p>
<h3> <span class="mw-headline" id="Phylogenetic_analysis_of_Enzymes"> Phylogenetic analysis of Enzymes </span></h3>
<p>In this lab we want to study the phylogenetic relationship of different enzymes.  We can align the amino acid sequence as well as the underlying nucleotide coding sequence. The functionality of a protein is reflected in its amino acid sequence; therefore the nucleotide sequence is usually converted into the amino acid sequence analogous to the translation. A point mutation in the DNA sequence does not necessarily lead to a change in the peptide chain because several different codons can be translated into the same amino acid. Conversely, a single nucleotide insertion might appear as a minor change on the DNA level, however, on the protein level, the nucleotide insertion can cause a shift in the whole codon pattern and therefore the sequence will be translated into a completely different amino acid sequence. Thus the amino acid sequences of the different proteins are compared with each other.<br/>
</p><p><br/>
</p>
<h3> <span class="mw-headline" id="Alignment_Scoring"> Alignment Scoring </span></h3>
<p>The sequences are aligned such that they reach the maximum similarity. The amino acid replacements are then rated according to their likelihood. For example a Gly-Ala change is more likely than a Gly-Pro change, which would alter the 3D structure of the protein. Thereby we can derive the degree of homology. These evolutionary distances are usually depicted as a phylogenetic tree. To validate the result, the analysis is repeated several times and the resulting trees are superimposed on each other (bootstrapping), showing only the phylogenetic tree that is the most likely. Each node is assigned a value based on the proportion of bootstrapped trees showing that same clade. Nodes with a value greater than 70% are considered as consistent. <br/>
The resulting phylogenetic tree provides insights on the taxonomy of the different species. In the case of proteins, the close homologs are usually responsible for similar functions. When well-studied proteins are included in the phylogenetic analysis, one can derive the biological function of closely related enzymes.
</p><p><br/>
</p><p><a href="/index.php/BLAST" title="BLAST">&#8656; BLAST</a>/ <a href="/index.php/Moss_Transformation" title="Moss Transformation"> Moss Transformation &#8658;</a><br/>
<a href="/index.php/PlantLab" title="PlantLab"> &#8656; Theory Overview</a>
</p>
<!-- 
NewPP limit report
Preprocessor node count: 11/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->

<!-- Saved in parser cache with key labster_com_db-Learn_:pcache:idhash:4187-0!*!0!!*!2!* and timestamp 20140813094951 -->
</div>				</body></html>