<html><head></head><body><div lang="en" dir="ltr" class="mw-content-ltr"><div class="thumb tright"><div class="thumbinner" style="width:444px;"><a href="/index.php/File:PTT.jpg" class="image"><img alt="" src="/images/thumb/PTT.jpg/442px-PTT.jpg" width="442" height="300" class="thumbimage"/></a>  <div class="thumbcaption">Figure 1. A schematic diagram showing the different steps of the protein truncation test.</div></div></div>
<p>A protein truncation test is a powerful method to view DNA mutation that results in protein truncation in vitro. Based on the impact to the translated proteins, DNA mutations are classified into frameshift mutation, nonsense mutation, missense (nonsynonymous) mutation, neutral mutation, and silent mutation. Nonsense mutation results in a premature stop codon. This means that the resulting protein will be terminated earlier than what it is supposed to be and can be detected using protein truncation test. This shorter protein is also called truncated protein. With protein truncation test, we do not need to use an animal as a model system to synthesize the protein; instead, we have an in vitro system where the resulting protein can be synthesized without the requirement of living cells.
</p><p><br/>
Protein truncation test is often used to analyze disease-related mutations, for example, in cancer where the mutation results in a nonfunctional protein. From protein truncation test, we can conclude whether there is a mutation in a particular part of the gene that results in protein truncation; however, we do not have information regarding where exactly the mutation is located and what is the resulting DNA sequence (for example, it could be a substitution creating a stop codon or a deletion resulting in a frameshift and then a stop codon or others). To find out more information regarding the mutation, we need to perform DNA sequencing as a validation method. With DNA sequencing, we will be able to know the exact sequence of DNA that results in a truncated protein. 
</p><p><br/>
Protein truncation test is composed of four major steps (Figure 1): 
</p>
<ol><li> Nucleic acid isolation; either genomic DNA, total RNA, or poly-A RNA.
</li><li> Amplification of a specific region of the gene of interest using PCR. In this step, a start codon (ATG) is added to the 5' end of the amplified DNA.
</li><li> In vitro transcription and translation of the product.
</li><li> Detection of the translated protein using polyacrylamide gel electrophoresis. 
</li></ol>
<p><br/>
</p>
<h2> <span class="mw-headline" id="Polyacrylamide_gel_electrophoresis">Polyacrylamide gel electrophoresis</span></h2>
<p>Polyacrylamide gel electrophoresis is a technique used to analyze proteins on the basis of their size. It is quite similar to <a href="/index.php/Gel_electrophoresis_MG" title="Gel electrophoresis MG">agarose gel electrophoresis</a> that is used to analyze DNA after performing PCR. Unlike DNA, proteins can be folded into a 3D structure. Before running them on the gel, we must first denature this 3D structure of the proteins and make them linear. We can do this by adding sodium dodecyl sulfate (SDS), which is a detergent that unravels proteins to make them linear. SDS is also required make proteins negatively charged. 
</p><p><br/>
When the proteins are linearized and have a negative charge, we can load them onto a polyacrylamide gel and apply electrical charge. The negatively charged proteins will travel through the porous polyacrylamide toward the positive pole, which is located at the end of the gel. The longer proteins will travel slower than the shorter proteins; this is how we can separate proteins on the basis of their length using polyacrylamide gel electrophoresis. In protein truncation test, the wild-type proteins will migrate lesser than the truncated proteins that are shorter. 
</p><p><br/>
<a href="/index.php/Gel_electrophoresis_MG" title="Gel electrophoresis MG">&#8656; Previous (Gel electrophoresis)</a> / <a href="/index.php/NGS_MG" title="NGS MG">Next (NGS) &#8658;</a>
</p><p><a href="/index.php/Medical_Genetics_case" title="Medical Genetics case">&#8656; Theory overview</a>
</p>
<!-- 
NewPP limit report
Preprocessor node count: 3/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->

<!-- Saved in parser cache with key labster_com_db-Learn_:pcache:idhash:3959-0!*!0!*!*!2!* and timestamp 20140814080741 -->
</div>				</body></html>