<html><head></head><body><div lang="en" dir="ltr" class="mw-content-ltr"><p>We recommend you to watch video about DNA profiling:
<a rel="nofollow" target="_blank" class="external free" href="http://vimeo.com/69239594">http://vimeo.com/69239594</a>
</p><p>Tandem repeats arise when a sequence of bases is repeated multiple times. The sequence ATGATGATGATGATGATGATGATG has the repeated sequence "ATG" eight times. The sequence being repeated is termed a <i>motif</i>. Repeats of 10-60 bases per repeat are known as minisatellites, where as shorter repeats (2-9 bases) are known as microsatellites (or STRs = Small Tandem Repeats). The tandem repeated regions are present at many sites in the human genome, and differ in their number of repeats.
</p>
<div class="thumb tright"><div class="thumbinner" style="width:363px;"><a href="/index.php/File:Tandem.jpeg" class="image"><img alt="" src="/images/thumb/Tandem.jpeg/361px-Tandem.jpeg" width="361" height="350" class="thumbimage"/></a>  <div class="thumbcaption">A tandem repeated region will have different number of repeats among individuals. Repeats can be dinucleotide, trinucleotide, and others. A set of primers placed on the flanking regions of a variable sequence will amplify fragments of different lengths depending on the number of repeats.</div></div></div>
<p><br/>
</p>
<h3> <span class="mw-headline" id="Tandem_Repeated_Regions_change_because_of_replication_errors">Tandem Repeated Regions change because of replication errors</span></h3>
<p>The difference in the number of tandem repeats in human arise because of errors in the replication of genomic DNA. The DNA polymerase slides along the DNA and adds complementary nucleotides. When moving over a highly repetitive sequence, such as a microsatellite, the DNA polymerase sometimes "hiccups," resulting either skipping an entire repeat or creating an extra repeat. <br/>
The process of adding or removing a single repeat of a motif is also called stepwise mutation process. DNA polymerase replication errors of highly repetitive sequences are fairly common. This implies that the mutation rate in microsatellites with a high number of repeats is very high.
</p><p><br/>
</p>
<h3> <span class="mw-headline" id="Tandem_Repeated_Regions_as_Molecular_Markers">Tandem Repeated Regions as Molecular Markers</span></h3>
<p>Microsatellites do not code for any protein, and are therefore considered to be neutral markers. High mutation rate and a large number of possible sites make microsatellites incredibly useful as molecular markers for population genetics, parentage assignment studies, and forensic science.
</p><p><br/>
</p><p><br/>
</p><p><a href="/index.php/DNA" title="DNA">&#8592; Previous (DNA)</a> / <a href="/index.php/DNA_Replication" title="DNA Replication">Next (DNA Replication) &#8594;</a>
</p><p><a href="/index.php/CSI_Case" title="CSI Case">&#8592; Theory overview</a>
</p>
<!-- 
NewPP limit report
Preprocessor node count: 7/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->

<!-- Saved in parser cache with key labster_com_db-Learn_:pcache:idhash:18-0!*!0!!*!2!* and timestamp 20140814080758 -->
</div>				</body></html>